workflows/quickstart/index.js

```diff
@@ -90,8 +88,16 @@ async function executeWorkflow(projectId, location, workflow) {
   }
 }
 
-executeWorkflow(projectId, location, workflowName).catch(err => {
+// [START workflows_api_quickstart_runtime_args]
+// Provide runtime arguments as a JSON string
+const runtimeArgs = searchTerm
+  ? JSON.stringify({searchTerm: searchTerm})
```

Comment:
Cleaner if you keep this logic within `executeWorkflow` and just have users pass in a search term string. If you do go this route, please add to the unit test a new condition.